internal/saucecloud/xcuitest.go

```diff
@@ -50,7 +50,16 @@ func (r *XcuitestRunner) RunProject() (int, error) {
 		return exitCode, err
 	}
 
-	passed := r.runSuites(appFileID, testAppFileID)
+	var otherAppsIDs []string
+	for _, o := range r.Project.Xcuitest.OtherApps {
+		ID, err := r.uploadProject(o, otherAppsUpload)
+		if err != nil {
+			return exitCode, err
+		}
+		otherAppsIDs = append(otherAppsIDs, fmt.Sprintf("storage:%s", ID))
+	}
```

Comment:
Since you pretty much have the exact same logic for both espresso and xcuitest for uploading multiple files, it might be worthwhile to add a convenience method `uploadProjects()` (notice the plural) that does exactly that.

Reply:
sure!